### PR TITLE
assets: Add a struct for storing stringified TLS assets

### PR DESCRIPTION
**Pull request by nhlfr**
Ref giantswarm/k8scloudconfig#80

### CODE REVIEW

**Review by rossf7 — APPROVED** [review]
LGTM thx <3

File: tls_assets.go [hunk 1]
```diff
@@ -87,3 +87,22 @@ func ValidComponent(el ClusterComponent, components []ClusterComponent) bool {
 	}
 	return false
 }
+
+// CompactTLSAssets is a struct used by operators to store and ren.
```
Comment (xh3b4sd) [inline tls_assets.go]: `and ren.`?

**Review by xh3b4sd — APPROVED** [review]
Please fix the comment. Then this makes sense to me.

File: tls_assets.go (hunk 1 above)
Comment (nhlfr) [reply tls_assets.go]: Good catch, fixed that

**Comment by asymmetric** [comment]
I'm confused about the usecase comprared to [this](https://github.com/giantswarm/aws-operator/blob/18bc88aed07c37fb6df6139a06b218ffe184ae03/service/create/encrypt.go#L15-L31).

Can someone please explain?

**Comment by xh3b4sd** [comment]
@asymmetric the aws-operator should probably make use of the structures provided by the certificatetpr.

**Pull request merged**